.github/workflows/release-java-sdk.yml

```diff
@@ -0,0 +1,31 @@
+name: Release Java SDK
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "The version of the Java SDK that you would like to release"
+        required: true
+        type: string
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Setup node
+        uses: actions/setup-node@v3
+
+      - name: Download Fern
+        run: npm install -g fern-api
```

Comment:
Can we add `fern-api` as a dev dependency to the main `package.json` and use that with `npm ci` so we can use the exact same version while publishing the packages, please?

Reply:
The CLI is smart enough to run itself on the version specified [here](https://github.com/metriport/metriport/blob/develop/fern/fern.config.json)

Reply:
As discussed offline, `fern-api` uses the version on `fern/fern.config.json` to determine the version, so no need for that.